runtime/onert/api/include/nnfw_experimental.h

```diff
@@ -220,6 +220,15 @@ typedef struct nnfw_train_info
   NNFW_TRAIN_OPTIMIZER opt = NNFW_TRAIN_OPTIMIZER_SGD;
 } nnfw_train_info;
 
+/**
+ * @brief Set train_info into session
+ * @note  This function should be called in MODEL_LOADED state,
+ *        after {@link nnfw_load_model_from_file} before {@link nnfw_prepare}
+ *
+ *  @return @c NNFW_STATUS_NO_ERROR If successful
+ */
+NNFW_STATUS nnfw_train_set_traininfo(nnfw_session *session, const nnfw_train_info *train_info);
+
```

Comment:
**This API can called after `model_loaded`.** 

Because.. If the user set train_info before the model loaded,
It is possible that train_info in the session is overwritten when the model is loaded.

So, To avoid unexpected overwritten while model loaded, Let's make sure that `nnfw_train_set_traininfo` called after model loaded.